File: BOJ/python3/1436.py
'''
영화감독 숌
https://www.acmicpc.net/problem/1436
'''
import sys
import time
start = time.time()  # 시작 시간
sys.stdin = open("input.txt", "r")  # input.txt 파일로 입력

N = int(input())

titles = []
for n in range(666, 2000000):
    if "666" in str(n):
        titles.append(n)

print(titles[N-1])
# 현재시각 - 시작시간 = 실행 시간(sec)
print("--------------------")
print(f"Elapsed Time: {round(time.time() - start, 3)}s")

File: BOJ/nodejs/1436.js
/*
영화감독 숌
boj.kr/1436
 */
const start = new Date().getTime();

const fs = require("fs");
const input = fs.readFileSync("input.txt").toString().split("\n");
//const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const N = 1 * input[0];

let n = 0,
  i = 665;
while (n < N) {
  if (i.toString().includes("666")) {
    n++;
  }
  i++;
}

const ans = i - 1;
console.log(ans);

const end = new Date().getTime();
console.log("--------------------");
console.log(`Elapsed Time: ${(new Date().getTime() - start) / 1000}s`);
